App/src/pages/RegularGroups.tsx

```diff
@@ -0,0 +1,83 @@
+import { Container, Table, Thead, Tbody, Tr, Th, Td } from "@chakra-ui/react";
+import { useContext, useEffect, useState } from "react";
+import { GetAllGroups, GroupDTO, MultiGroupResponseDTO } from "../services/group";
+import { UserContext } from "../App";
+import { UserDTO } from "../services/user";
+import Header from "../shared/Header";
+
+export default function DisplayGroups() {
+    const [myGroups, setMyGroups] = useState<GroupDTO[]>([]);
+    const user = useContext(UserContext);
+    const userDTO: UserDTO = { firstName: user.firstName, id: "7", lastName: user.lastName, email: user.email };
```

Comment:
This line is not needed. You've already extracted the currently logged in user into the `user` variable.

Reply:
Alright, it was originally there to solve issues with membership but the version you proposed seems to have fixed that.